js/dogs.js

```diff
@@ -0,0 +1,71 @@
+
+const url = 'https://api-staging.adoptapet.com/search/pet_search?key=hg4nsv85lppeoqqixy3tnlt3k8lj6o0c&v=3&output=json&city_or_zip=29445&geo_range=50&species=dog&breed_id=real=801&sex=m&age=young&color_id=54&pet_size_range_id=2&hair=&bonded_pair=&special_needs=&include_mixes=&added_after=&start_number=1&end_number=50&meta_only=0';
```

Comment:
Just as a best practices comment: When you're using an API you never want to store your API key in a public repo. Instead, you'd want to store them securely using something like [GitHub Secrets](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions) or [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).